docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -0,0 +1,156 @@
+
+If you don't want to use the Teams Toolkit and your project wasn't created with the Office Yeoman generator, use the office-addin-manifest-converter tool.
+
+In the root of the project, open a command prompt or bash shell and run the following command. This command puts the unified manifest in a subfolder with the same name as the filename stem of the original XML manifest. For example, if the manifest is named MyManifest.xml, the unified manifest is created at .\MyManifest\MyManifest.json. For more details about this command, see [Office-Addin-Manifest-Converter](https://www.npmjs.com/package/office-addin-manifest-converter).
+
+```command&nbsp;line
+npx office-addin-manifest-converter convert -m <relative-path-to-XML-manifest>
+```
+
+Once you have the unified manifest created, there are two ways to create the zip file and sideload it. They are described in the next two subsections.
+
+#### Sideload with the Office-Addin-Debugging tool
+
+1. To sideload the add-in, run the following command. This command puts the unified manifest and two default icon image files into a zip file and sideloads it to the Office application. It also starts a server in a separate NodeJS window to host the add-in files on localhost. Note that you pass the path to the *unified manifest* that you created in the previous step. For more details about this command, see [Office-Addin-Debugging](https://www.npmjs.com/package/office-addin-debugging).
+
+    ```command&nbsp;line
+    npx office-addin-debugging start <relative-path-to-unified-manifest> desktop
+    ``` 
+
+1. When you use office-addin-debugging to start an add-in, *always stop the session with the following command*. Closing the server window doesn't reliably stop the server and closing the Office application doesn't reliably cause Office to unacquire the add-in.
+
+    ```command&nbsp;line
+    npx office-addin-debugging stop <relative-path-to-unified-manifest>
+    ``` 
+
+#### Sideload with the Teams Toolkit CLI
+
+1. Manually create the zip package with these steps:
+
+    1. Open the unified manifest and scroll to the "icons" property. Note the relative path of the two image files.
+    1. Use any zip utility to create a zip file that contains the unified manifest and the two image files. *The image files must have the same relative path in the zip file as they do in the project.* For example, if the relative path is "assets/icon-64.png" and "assets/icon-128.png", then you must include the "assets" folder with the two files in the zip package. 
+    1. If the folder contains other files, such as image files used in the Office ribbon, remove these from the zip package. It should have only the two image files specified in the "icons" property (in addition to the manifest in the root of the zip package). 
+
+1. In the root of the project, open a command prompt or bash shell and run the following commands.
+
+    ```command&nbsp;line
+    npm install -g @microsoft/teamsfx-cli@1.0.6-alpha.df7cb43be.0
```

Comment:
I believe current versions of teamsfx-cli support this now so the alpha version isn't needed anymore.